scripts/cloud-api-demo/reset_database.py

```diff
@@ -0,0 +1,63 @@
+import argparse
```

Comment:
This must be the first line to specify the specific interpreter to use

```
#!/usr/bin/env python3
```

Reply:
Added